src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
   margin: 0;
+
+  --text-color-second: #00acdc;
```

Comment:
Good job using a CSS variable for the blue color. This aligns with the task requirement to use CSS variables.